assets/js/phoenix_live_view/live_socket.js

```diff
@@ -635,6 +635,9 @@ export default class LiveSocket {
       if(capture){
         target = e.target.matches(`[${click}]`) ? e.target : e.target.querySelector(`[${click}]`)
       } else {
+        // a synthetic click event (detail 0) will not have caused a mousedown event,
+        // therefore the clickStartedAtTarget is stale
+        if(e.detail === 0) this.clickStartedAtTarget = e.target
```

Comment:
we could also add a listener for `mouseup` and unset `this.clickStartedAtTarget`, e.detail is not supported on Internet Explorer, but I guess that should not matter any more 😃